apps/rest_api/test/controllers/admin/user_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule RestAPI.Controllers.Admin.User do
+  use RestAPI.ConnCase, async: true
+
+  alias RestAPI.Ports.{AuthenticatorMock, ResourceManagerMock}
+  alias ResourceManager.Identity.Commands.Inputs.CreateUser
+
+  @create_endpoint "/admin/v1/users"
+
+  describe "POST #{@create_endpoint}" do
+    test "should render user identity response", %{conn: conn} do
+      password = "MyP@ssword"
+
+      params = %{
+        "username" => "Shurato",
+        "password" => password,
+        "scopes" => [
+          "6a3a3771-9f56-4254-9497-927e441dacfc",
+          "8a235ba0-a827-4593-92c9-6248bef4fa06"
+        ]
+      }
+
+      expect(AuthenticatorMock, :generate_hash, fn password_to_hash, :argon2 ->
+        assert password = password_to_hash
+        "password_hashed"
+      end)
+
+      expect(ResourceManagerMock, :create_identity, fn input ->
+        assert is_map(input)
+
+        {:ok,
+         %{
+           id: Ecto.UUID.generate(),
+           inserted_at: NaiveDateTime.utc_now(),
+           is_admin: false,
+           status: "active",
+           updated_at: NaiveDateTime.utc_now(),
+           username: "Shurato"
+         }}
+      end)
+
+      assert %{
+               "id" => _id,
+               "inserted_at" => _inserted_at,
+               "is_admin" => false,
+               "status" => "active",
+               "username" => "Shurato"
+             } =
+               conn
+               |> post(@create_endpoint, params)
```

Comment:
As it's an authenticated request we have to pass the `access_token`` on authorization header here (after removing the comment on router).